stream-compositions/cursors/transaction-cursor/src/main/resources/db/changelog/db.changelog-1.0.0.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+    <changeSet id="initial_001" context="initial" author="backbase">
+        <comment>create the txn_cursor table</comment>
+        <createTable tableName="txn_cursor" remarks="Table to store the transaction cursor">
+            <column name="id" type="varchar(36)">
```

Comment:
Since arrangement_id column is already not null & unique, why not use arrangement_id as a primary key?